apps/video/test/unit/video_test.js

```diff
@@ -1815,6 +1892,7 @@ suite('Video App Unit Tests', function() {
     test('#updateSelection: update (remove) selected filenames', function() {
```

Comment:
Again, I don't see any difference between this test and the second updateSelection test that tests a thumbnail that was previously selected is deselected. If you also see no difference, it would be great to combine the two into one test.